api/src/data/migrations/003.create-department.ts

```diff
@@ -0,0 +1,13 @@
+import * as knex from "knex";
+
+exports.up = async function(knex: knex.Knex, Promise: any) {
+    await knex.schema.createTable("department", function(table) {
```

Comment:
@burkkyy Please make all table names follow standard English plurals.

(mostly just adding here so you can see the comment setup in GitHub)

Reply:
Many ORMs will use https://www.npmjs.com/package/inflection, or their language equivalent, for mapping table names.